### PR TITLE
Get and Count method with few corrections

### PR DESCRIPTION
**Pull request by Joshua-Enrico**
here we add get and count method , also tests added.

i found and error on amenity test file that has been fixed and finally i rebuild basemodel method in order to optimize execution because some test were failing because of that.
do not hesitate to ask any questions.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: Continuous Integration
+on: [push]
+jobs:
```
Comment (AlexisCORAN) [inline .github/workflows/main.yml:3]: file

**Review by AlexisCORAN — COMMENTED** [review]
Added files

File: .github/workflows/main.yml (hunk 1 above)
Comment (Joshua-Enrico) [reply .github/workflows/main.yml:3]: testing

**Review by Joshua-Enrico — COMMENTED** [review]
testing

File: models/base_model.py [hunk 2]
```diff
@@ -58,7 +51,7 @@ def save(self):
         models.storage.new(self)
         models.storage.save()
 
-    def to_dict(self):
+    def to_dict(self, secure_pwd=True):
```
Comment (AlexisCORAN) [inline models/base_model.py:54]: Why was the parameters of this function updated?

**Review by AlexisCORAN — COMMENTED** [review]
Test added correctly the tests for the files in the engine directory and and the corresponding requirements.

File: models/base_model.py (hunk 2 above)
Comment (Joshua-Enrico) [reply models/base_model.py:54]: that are requirements for task 15 , is not finished

**Pull request merged**